evm/src/cpu/kernel/asm/core/syscall_stubs.asm

```diff
@@ -0,0 +1,54 @@
+// Labels for unimplemented syscalls to make the kernel assemble.
+// Each label should be removed from this file once it is implemented.
+
+global sys_sdiv:
+global sys_smod:
+global sys_signextend:
+global sys_slt:
+global sys_sgt:
+global sys_byte:
```

Comment:
I think it would be good to have some native impl of `BYTE`, maybe using the logic table, as the kernel uses it quite a bit. But it's fine for now

Reply:
Sure! `BYTE` should not be difficult to implement. I’ll add a flag for it!